### DISCUSSION TITLE
VHS Audio effect

### DISCUSSION THREAD

**Discussion by SharkGitHubz** [ntsc-rs/ntsc-rs#244] [Ideas]
Basically, whenever you make a video on **NTSC-RS**, a VHS Audio effects gets slapped onto the video automatically, you can tweak it like how much noise you want and more.

**Comment by valadaptive** [comment]
I'm tracking this in https://github.com/valadaptive/ntsc-rs/issues/15. It's a huge undertaking, and there's already a pretty good tape audio effect (Chow Tape) already.
